runtime/oti/j9nonbuilder.h

```diff
@@ -3396,6 +3397,9 @@ typedef struct J9Class {
 	/* A linked list of weak global references to every resolved MemberName whose clazz is this class. */
 	J9MemberNameListNode *memberNames;
 #endif /* defined(J9VM_OPT_OPENJDK_METHODHANDLE) */
+#if defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES)
+	struct J9Class* nullRestrictedArrayClass;
```

Comment:
The `*` should be right aligned to match the other members, same comment for `companionArray`.